index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/60f96566f6.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <img  src="elements/my-logo.png" alt="">
+        <nav>
+            <i class="fas fa-bars"></i>
+        </nav>
+    </header>
+    <main>
+        <img id="ellipse-11" src="elements/Ellipse-11.jpg" alt="">
+        <img id="ellipse-05" src="elements/Ellipse-5.png" alt="">
+        <img id="rect-07" src="elements/Rectangle-7.png" alt="">
+        <img id="ellipse-03" src="elements/Ellipse-3.png" alt="">
+        <img id="ellipse-12" src="elements/Ellipse-12.png" alt="">
+        <img id="ellipse-04" src="elements/Ellipse-4.png" alt="">
+        <div id="wrapper">
+            <p id="primary-text">I'm Enmanuel, glad to see you!</p>
+            <p id="secondary-text">Im a software!, I can help you build a product, feature or website look through some of my work and experience! 
+                If you like what you see and have a project you need coded,don't hesitate to contact me.</p>
+            <p id="third-text">let's connect</p>
+            <div id="social">
+                <img src="elements/twitter-icon.png" alt="">
+                <img src="elements/github-icon.jpg" alt="">
```

Comment:
alt attribute should not be blank , please add the image name in the alt attribute
icons should be in  a list and wrapped in a tag